packages/addons/service/fd628/changelog.txt

```diff
@@ -0,0 +1,2 @@
+1.0.100
```

Comment:
We avoid major versioning as it results in loads of makework to go around changing them all when we bump repo versions, so this should be just `100` not `9.0.100` or `1.0.100`

Reply:
Updated, thanks